<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThemeSettingsDialog</class>
 <widget class="QDialog" name="ThemeSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Theme Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTableWidget" name="table">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>11</number>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <row/>
       <column>
        <property name="text">
         <string>Setting</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Color</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>Comments</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>Doc Comments</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="3" column="0">
        <property name="text">
         <string>Numbers</string>
        </property>
       </item>
       <item row="3" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="4" column="0">
        <property name="text">
         <string>Keywords</string>
        </property>
       </item>
       <item row="4" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="5" column="0">
        <property name="text">
         <string>Strings</string>
        </property>
       </item>
       <item row="5" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="6" column="0">
        <property name="text">
         <string>Preprocessors</string>
        </property>
       </item>
       <item row="6" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="7" column="0">
        <property name="text">
         <string>Operators</string>
        </property>
       </item>
       <item row="7" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="8" column="0">
        <property name="text">
         <string>Unclosed Strings</string>
        </property>
       </item>
       <item row="8" column="1">
        <property name="text">
         <string/>
        </property>
       </item>
       <item row="9" column="0">
        <property name="text">
         <string>Doc Keywords</string>
        </property>
       </item>
       <item row="10" column="0">
        <property name="text">
         <string>Doc Numbers</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::ExpandingFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Background Color:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="ColorBox" name="backgroundColorBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Font:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QFontComboBox" name="fontBox"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Font Size:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="fontSizeBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>26</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="reset">
       <property name="text">
        <string>Reset Defaults</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="ui_buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ColorBox</class>
   <extends>QLineEdit</extends>
   <header>color_box.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ThemeSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ui_buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ThemeSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
